fix(tuto): tutos + iot dates MTA-1688d (#469)
scaleway/docs-content: tutorials/zabbix-monitoring/index.mdx

  <Lightbox src="scaleway-zabbix_44.webp" alt="" />

  A check of pre-requirements displays, all off the values must have the status **OK**.

2. Click **Next step** to continue:

  <Lightbox src="scaleway-zabbix_install_02.webp" alt="" />

3. Enter the MariaDB (MySQL) database information. They are already configured for the Zabbix server, but the web interface requires them too. Click **Next step** once the information is filled in the form.

  <Lightbox src="scaleway-zabbix_install_03.webp" alt="" />

4. Click **Next step** to continue. A pre-installation summary displays.

  <Lightbox src="scaleway-zabbix_install_04.webp" alt="" />

> 5. Check that all given information are correct and click **Next step** to continue.

Reviewer: ```suggestion
5. Check that all given information is correct and click **Next step** to continue.
```

  Uncomment the last line of block and set the path to the PSK file: `TLSPSKFile=/etc/zabbix/zabbix_agentd.psk`. Then save the file and exit the editor.

12. Restart the `zabbix-agent` service and enable it to start automatically during the system boot.

  ```
  systemctl restart zabbix-agent
  systemctl enable zabbix-agent
  ```

> 13. Allow traffic on port 10050, in case ufw is installed and active on the instance.

Reviewer: ```suggestion
13. Allow traffic on port 10050, if ufw is installed and active on the Instance.
```
